scss/civicrm/common/_tooltip.scss

```diff
@@ -1,114 +1,103 @@
-.crm-tooltip-wrapper {
-  background-image: none !important;
-  overflow: visible;
-  padding-bottom: 0;
-
-  .crm-tooltip {
-    @include box-shadow($box-shadow);
-    background: $crm-white !important;
-    border-radius: $border-radius-base;
-    padding: 0;
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
+#crm-container {
```

Comment:
@mukeshcompucorp i really feel that this kind of changes should be disclosed in some "Technical details" section in the PR, we can't always wait for me to ask about them. Please let's have a habit of spotting changes that are meaningful from a technical point of view and describe them directly in the PR